idx-template.nix

```
@@ -0,0 +1,23 @@
+{pkgs, branch ? "loic-sharma/decorators", repo ? "https://github.com/loic-sharma/flutter", ...}: {
```

Ah and since we don't have a remote here, branch should just be `decorators`

```suggestion
{pkgs, branch ? "decorators", repo ? "https://github.com/loic-sharma/flutter", ...}: {
```